app.rb

```diff
@@ -0,0 +1,95 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+    end
+  end
+
+  def create_person(type, age, name)
+    if type == 'student'
+      classroom = ask_for_classroom
+      person = Student.new(age, classroom, name)
+    elsif type == 'teacher'
+      specialization = ask_for_specialization
+      person = Teacher.new(age, specialization, name)
+    else
+      puts 'Invalid person type.'
+      return
+    end
+    @people << person
+    puts "Person created: Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+  end
+
+  def create_book(title, author)
+    book = Book.new(title, author)
+    @books << book
+    puts "Book created: Title: #{book.title}, Author: #{book.author}"
+  end
+
+  def create_rental(person_id, book_title, date)
+    person = find_person_by_id(person_id)
+    book = find_book_by_title(book_title)
+
+    if person && book
+      rental = Rental.new(date, book, person)
+      @rentals << rental
+      puts "Rental created: Book: #{book.title}, Person: #{person.name}, Date: #{rental.date}"
+    else
+      puts 'Person or book not found.'
+    end
+  end
+
```

Comment:
- [ ] Thank you for addressing the reviewer's concern. As per the project requirements, it is important that the console app behaves in the same way as demonstrated in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). To create a rental, the books to be rented should be listed and selected by index, followed by the persons who will be renting the books, also selected by index, just as shown in the example video. Your app functions differently at the moment, so kindly ensure that it behaves in the same way as demonstrated in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ).  :pray: .